server/graphql/types/src/types/vaccine_course.rs

```diff
@@ -40,8 +40,8 @@ impl VaccineCourseNode {
         self.row().coverage_rate
     }
 
-    pub async fn is_active(&self) -> bool {
-        self.row().is_active
+    pub async fn use_in_gaps_calculations(&self) -> bool {
+        self.row().use_in_gaps_calculations
     }
```

Comment:
I feel like this is a safe-enough breaking API change - but could deprecate instead if we think this might be more widely queried?

Reply:
This will all get more fun with the Cambrian explosion of backend plugins